push/sender/pom.xml

```diff
@@ -130,6 +130,54 @@
             <artifactId>java-adm</artifactId>
             <version>0.1.0</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.wildfly.extras.creaper</groupId>
+            <artifactId>creaper-core</artifactId>
+            <version>0.9.2</version>
```

Comment:
1.0.0 is out, and on maven central http://search.maven.org/#search%7Cga%7C1%7Ccreaper-core